lib/data/data_sources/remote_data_source/models/baseMessage.dart

```diff
@@ -11,12 +12,14 @@ SendbirdSdk sendbird = locator.get();
 extension BaseMessageExtension on BaseMessage {
   ChatMessage toDomain() => ChatMessage(
         createdAt: this.createdAt != null ? this.createdAt : 0,
+        requestId: this.requestId,
         data: this.data,
         messageId: this.messageId != null ? this.messageId : null,
```

Comment:
here instead of asking this, you can send messageId directly because if it's null you're gonna send null